sailfishusermanager.cpp

```diff
@@ -219,8 +219,8 @@ uint SailfishUserManager::addUser(const QString &name)
                 count++;
         }
     }
-    if (count > SAILFISH_USERMANAGER_MAX_USERS) {
-        // Just > above because master user is also member of users group
+    if (count > (SAILFISH_USERMANAGER_MAX_USERS - 1)) {
+        // Luks has 8 slots where one is reserved for backup and one for master
```

Comment:
This comment about number of slots feels a bit out of place. Wouldn't it be better to locate that information next to the constant definition?